src/test/java/org/broadinstitute/hellbender/cmdline/PicardProgramExecutorIntegrationTest.java

```diff
@@ -1,53 +1,64 @@
-package org.broadinstitute.hellbender.tools.picard.fastq;
+package org.broadinstitute.hellbender.cmdline;
 
 import org.broadinstitute.hellbender.CommandLineProgramTest;
 import org.broadinstitute.hellbender.utils.test.IntegrationTestSpec;
-import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
 import org.testng.annotations.Test;
 
 import java.io.File;
 import java.io.IOException;
+import java.util.List;
 
-public final class NormalizeFastaIntegrationTest extends CommandLineProgramTest {
 
+// Test that we can successfully run Picard tools.
+public final class PicardProgramExecutorIntegrationTest extends CommandLineProgramTest {
+
+    // Use the Picard tool "NormalizeFasta" as a simple test case.
     private static final File TEST_DATA_PATH = new File(getTestDataDir(), "picard/fastq/NormalizeFasta");
 
     @Override
     public String getTestedClassName() {
-        return NormalizeFasta.class.getSimpleName();
+        return picard.reference.NormalizeFasta.class.getSimpleName();
+    }
+
+    @Override
+    public List<String> injectDefaultVerbosity(final List<String> args) {
+        // override/suppress GATK-specific argument injection
+        return args;
     }
 
     @Test
-    public void testNormalize() throws IOException {
+    public void testPicardNormalizeFasta() throws IOException {
         final File input = new File(TEST_DATA_PATH, "testfasta.fasta");
         final File expectedFile = new File(TEST_DATA_PATH, "testFASTA_WS_4.fasta");
-        final File outfile = BaseTest.createTempFile("normalized", ".fasta");
+        final File outfile = createTempFile("normalized", ".fasta");
 
         final String[] args = {
-                "--input", input.getAbsolutePath(),
-                "--output", outfile.getAbsolutePath(),
+                "-I", input.getAbsolutePath(),
+                "-O", outfile.getAbsolutePath(),
                 "--TRUNCATE_SEQUENCE_NAMES_AT_WHITESPACE", "TRUE",
                 "--LINE_LENGTH", "5",
         };
 
-        runCommandLine(args);
+        Assert.assertEquals(runCommandLine(args), 0);
 
         IntegrationTestSpec.assertEqualTextFiles(outfile, expectedFile);
     }
 
-    @Test(expectedExceptions = IllegalArgumentException.class)
-    public void testFailNormalize() {
+    @Test
+    public void testPicardNormalizeFastaWithBadArgs() throws IOException {
         final File input = new File(TEST_DATA_PATH, "testfasta.fasta");
+        final File outfile = createTempFile("normalized", ".fasta");
 
+        // use GATK-style lower case names, which are rejected by Picard
```

Comment:
Need slightly clearer explanation here of why this fails (eg., mention explicitly that the Picard arg is named `--OUTPUT`)